src/Diagrams/Swimunit/Dotmatrix.hs

```diff
@@ -0,0 +1,843 @@
+{-# LANGUAGE NoMonomorphismRestriction #-}
+
+module Diagrams.Swimunit.Dotmatrix
+where
+
+import qualified Data.Map as Map
+
+import Diagrams.Prelude
+import Diagrams.Backend.SVG.CmdLine
```

Comment:
`contrib` modules shouldn't depend on a specific backend. Instead of `Diagram B` it should be something like `TypeableFloat n => QDiagram b V2 n Any -> Dotfont -> String -> QDiagram b V2 n Any`.